CONTRIBUTING.md

```diff
@@ -1,13 +1,25 @@
-# Request for contributions
-We are always looking for contributions to the Open Build Service. Read this guide on how to do that. 
+# <a name="toc">Table of Contents
+
+1. [Request for contributions](#request)
+2. [How to contribute code](#contribute_code)
+3. [How to contribute issues](#contribute_issues)
+4. [Contribute to the OBS documentation](#contribute_docu)
+5. [Conduct](#conduct)
+6. [Communication](#communication)
+7. [Rubocop](#rubocop)
+8. [Setup an OBS backend for development](#setup_backend)
+9. [Quick Start Guid (Howto setup a developer VM)](#quick_start)
+
+# <a name="request"/> Request for contributions
```

Comment:
If you move your mouse left of the title you see the real anchor. It's lowercase and with hyphens :-)